CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the IBM® Db2® Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Exit code 1 when DB2 calls do not succeed
```

Comment:
We should explain the actual fix for this. Also, this needs a PR/issue link.

Reply:
Updated link and wording to state that the command will exit with code 1 if something goes wrong with issuing commands. This tells automation that something went wrong.